tests/unit/verifiers/test_base.py

```diff
@@ -200,48 +203,47 @@ def test_base_verifier_run_action_on_all_units(mock_run_action_on_units, model):
 
 def test_base_verifier_check_has_sub_machines(mocker):
     """Test check unit has sub machines verifier."""
-    main_unit = 'nova-compute/0'
-    child_unit = 'child-unit/0'
+    main_unit = "nova-compute/0"
+    child_unit = "child-unit/0"
     # Mock async lib calls
     loop = MagicMock()
-    mocker.patch.object(asyncio, 'get_event_loop').return_value = loop
+    mocker.patch.object(asyncio, "get_event_loop").return_value = loop
 
     # Generic mock for all units
     model = Model()
-    unit_data = {'subordinate': False}
-    mocker.patch.object(Unit, 'data',
-                        new_callable=PropertyMock(return_value=unit_data))
+    unit_data = {"subordinate": False}
+    mocker.patch.object(Unit, "data", new_callable=PropertyMock(return_value=unit_data))
 
     # dict of units/machines to test with
-    mocker.patch.object(Model, 'units')
+    mocker.patch.object(Model, "units")
     units = [
-        {'name': main_unit, 'machine': '0', 'leader': True},
-        {'name': child_unit, 'machine': '0/lxd/0', 'leader': True},
+        {"name": main_unit, "machine": "0", "leader": True},
+        {"name": child_unit, "machine": "0/lxd/0", "leader": True},
     ]
 
     # is_leader_from_status result for every child
-    mocker.patch.object(loop, 'run_until_complete').return_value = [True]
+    mocker.patch.object(loop, "run_until_complete").return_value = [True]
 
     unit_list = []
     for unit in units:
-        unit['unit_object'] = Unit(unit['name'], model)
-        unit_list.append((unit['name'], unit['unit_object']))
+        unit["unit_object"] = Unit(unit["name"], model)
+        unit_list.append((unit["name"], unit["unit_object"]))
         mocker.patch.object(
-            unit['unit_object'], 'machine',
+            unit["unit_object"],
+            "machine",
             new_callable=PropertyMock(return_value=MagicMock()),
         )
-        mocker.patch.object(
-            unit['unit_object'].machine, 'entity_id', unit['machine']
-        )
+        mocker.patch.object(unit["unit_object"].machine, "entity_id", unit["machine"])
 
-    mocker.patch.object(Model.units, 'items').return_value = unit_list
+    mocker.patch.object(Model.units, "items").return_value = unit_list
 
-    expected_partial_result = Partial(Severity.WARN,
-                                      f'{main_unit} has units running on child '
-                                      f'machines: {child_unit}*')
+    expected_partial_result = Partial(
+        Severity.WARN,
+        f"{main_unit} has units running on child " f"machines: {child_unit}*",
```

Comment:
strings can be merged

Reply:
I manually looked for `" "` sting and `' '`, but for `" f"` :laughing: